src/tests/unit/tests/common/components/cards/__snapshots__/rule-resources.test.tsx.snap

```diff
@@ -5,183 +5,287 @@ exports[`RuleResources renders with {
   guidanceLinks: [ [length]: 0 ],
   linkComponent: 'ExternalLink'
 } 1`] = `
-<div
-  className="ruleMoreResources"
->
+<DocumentFragment>
   <div
-    className="moreResourcesTitle"
+    class="ruleMoreResources"
   >
-    Resources for this rule
-  </div>
-  <span
-    className="ruleDetailsId"
-  >
-    <ExternalLink
-      href="test-url"
+    <div
+      class="moreResourcesTitle"
     >
-      More information about 
-      some-rule
-    </ExternalLink>
-  </span>
-  <span
-    className="ruleGuidance"
-  >
-    <GuidanceLinks
-      LinkComponent={[Function]}
-      links={[]}
-    />
-    <GuidanceTags
-      deps={
-        {
-          "LinkComponent": [Function],
-        }
-      }
-      links={[]}
-    />
-  </span>
-</div>
+      Resources for this rule
+    </div>
+    <span
+      class="ruleDetailsId"
+    >
+      <div
+        class="ms-TooltipHost insightsTooltipHost root-109"
```

Comment:
It might be worth it to mock the `ExternalLink` component to cut down on superfluous HTML in this snapshot.

Reply:
Yes, we have mocked **HeadingElementForLevel** component as per suggestion.